src/workers/parquet-exporter.ts

```diff
@@ -114,36 +136,59 @@ export class ParquetExporter {
             durationInSeconds,
           });
 
-          this.isExporting = false;
-          this.worker?.terminate();
```

Comment:
I stop terminating the worker directly from here because it runs before it closes the db and delete the files.